draft-gpew-priv-ppm.md

```diff
@@ -792,57 +802,57 @@ usually need to be encrypted in order to protect user privacy. However, the
 details of precisely how the state is encrypted and the information that it
 carries is up to the helper implementation.
 
-### Output Share Request {#output-share-request}
+### Aggregate Share Request {#aggregate-share-request}
 
 Once the aggregators have verified at least as many reports as required for the
-PPM task, the leader issues an *output share request* to each helper. The helper
-responds to this request by extracting its output share from its state and
-encrypting it under the collector's HPKE public key.
+PPM task, the leader issues an "aggregate-share request" to each helper. The
+helper responds to this request by extracting its aggregate share from its state
+and encrypting it under the collector's HPKE public key.
 
 For each aggregator endpoint `[aggregator]` in the parameters associated with
 `CollectReq.task_id` (see {{pa-collect}}) except its own, the leader sends a
-POST request to `[aggregator]/output_share` with the following message:
+POST request to `[aggregator]/aggregate_share` with the following message:
 
 ~~~
 struct {
   TaskID task_id;
   Interval batch_interval;
   opaque helper_state<0..2^16>;
-} OutputShareReq;
+} AggregateShareReq;
 ~~~
 
 * `task_id` is the task ID associated with the PPM parameters.
 * `batch_interval` is the batch interval of the request.
 * `helper_state` is the helper's state, which is carried across requests from
   the leader.
 
-To respond to an output share request, the helper first looks up the PPM
+To respond to an AggregateShareReq message, the helper first looks up the PPM
 parameters associated with task `task_id`. Then, using the procedure in
 {{batch-parameter-validation}}, it ensures that the request meets the
-requirements of the batch parameters. If so, it aggregates all valid input
-shares that fall in the batch interval into an output share. The response
+requirements of the batch parameters. If so, it aggregates all valid output
+shares that fall in the batch interval into an aggregate share. The response
 contains an opaque, VDAF-specific message:
 
 ~~~
 struct {
   opaque message<0..2^16-1>; // VDAF message
-} OutputShare;
+} AggregateShare;
 ~~~
 
-Next, the helper encrypts the output share `output_share` under the collector's
+Next, the helper encrypts the aggregate share `agg_share` under the collector's
 public key as follows:
 
 ~~~
 enc, context = SetupBaseS(pk,
-                          "pda output share" || task_id || server_role)
-encrypted_output_share = context.Seal(batch_interval, output_share)
+                          "pda agg share" || task_id || server_role)
```

Comment:
Is "aggregate" not gonna fit?

Reply:
So I'm thinking it might simplify analysis if we make the info string the same length for each invocation of HPKE the same length. I'm thinking "ppm inp share" for the encrypted input shares and "ppm agg share" for the encrypted aggregate shares. I sort of half-assed it here, but I was planning on doing this in a future PR. I'll just change it to "aggregate" in this PR and defer that discussion.

Reply:
Done